testing-with-jest.test.js

```diff
@@ -9,19 +9,73 @@
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
 
+describe("addThemUpNo", () => {
+    test( "add two numbers" , () => {
+      expect(addThemUp()).toEqual(16)
+        })
+})
+
+var numb1 = 10
+var numb2 = 5
+
+const addThemUpNo =  () => {
+    { return numb1 + numb2} 
+}
 
 
 
 // --------------------1b) See the test fail. THEN write the code to make the test pass.
 
+//previous code just have a wrong return to make sure the function is working
+
+describe("addThemUp", () => {
+    test( "add two numbers" , () => {
+      expect(addThemUp()).toEqual(15)
+        })
+})
 
+var numb1 = 10
+var numb2 = 5
+
+const addThemUp =  () => {
+    { return numb1 + numb2} 
+}
```

Comment:
This section, as the formatting implies, is where the actual code should have been for your function. You've just copy pasted the same test and code from above. 

Remember the steps to TDD.
First: Write the test
Second: See the test fail.
Third: Write the code.
Fourth: See the test pass.

When running these tests at the end, did the test pass?